am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -155,6 +160,13 @@ public FilterResourceResponse filterResource(@NotBlank String userId,
         List<ExplicitAccessRecord> explicitAccess = jdbi.withExtension(AccessManagementRepository.class,
             dao -> dao.getExplicitAccess(userId, resource.getResourceId()));
 
+        List<Map<JsonPointer, Set<Permission>>> permissionsForRelationships = explicitAccess.stream()
```

Comment:
Seems like variable could be moved down to keep it close the the place it is actually used.

Reply:
👍